package.json

```diff
@@ -15,21 +15,24 @@
     "posttest": "npm run lint && npm run test:git-history"
   },
   "devDependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/polyfill": "^7.2.5",
-    "@commitlint/cli": "^8.2.0",
-    "@commitlint/config-conventional": "^8.2.0",
-    "amex-jest-preset-react": "^5.0.1",
-    "enzyme": "^3.9.0",
+    "@babel/cli": "^7.8.3",
+    "@babel/core": "^7.8.3",
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
+    "amex-jest-preset-react": "^5.0.2",
+    "babel-preset-amex": "^2.1.0",
+    "core-js": "^3.6.4",
```

Comment:
is core-js a direct dependency or is it brought in via jest or maybe amex-jest-preset-react?

Reply:
🤦‍♂️sorry, was thinking about jsdom due to some slack messages
core-js is babel related